test/index.test.js

```diff
@@ -50,13 +50,41 @@ describe('stylelint-webpack-plugin', function () {
   });
 
   it('sends warnings properly', function () {
-    return pack(assign({}, baseConfig, { context: path.resolve('./test/fixtures/rule-warning') }))
+    return pack(assign({}, baseConfig(), { context: path.resolve('./test/fixtures/rule-warning') }))
       .then(function (stats) {
         expect(stats.compilation.errors).to.have.length(0);
         expect(stats.compilation.warnings).to.have.length(1);
       });
   });
 
+  it('pushes errors using the webpack formatter', function () {
```

Comment:
Please make a `context('using the webpack formatter', function () { ... })` to group these tests.